README.md

```diff
@@ -0,0 +1,24 @@
+# querystring
+
+[![GitHub Actions Status](https://github.com/Gozala/querystring/workflows/CI/badge.svg)](https://github.com/Gozala/querystring/actions)
+[![GitHub Legacy Actions Status](https://github.com/Gozala/querystring/workflows/LEGACY/badge.svg)](https://github.com/Gozala/querystring/actions)
```

Comment:
I think this belongs to PR which configures the actions (?)

Reply:
Yes this was supposed to be in GitHub Actions branch but since `Readmd.md` is changed to `README.md` in this one I thought of adding it here so there can be minimal diff, so first CI branch needs to get merged I guess

Reply:
Ok, let's put this one on hold then for now

Reply:
Done